src/Entities/Campaigns/Delete.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Digitonic\PassonaClient\Entities\Campaigns;
+
+use Digitonic\PassonaClient\Requests\BaseRequest;
+
+class Delete extends BaseRequest
+{
+    const ENDPOINT = 'campaigns/{campaignUuid}';
+
+    /**
+     * @return string
+     */
+    protected function getFullEndpoint(): string
```

Comment:
Just a consideration why is this not default in the baserequest? same for the requiresEntityUuid if most of the time is going to be true why not just use it as default?